### DISCUSSION TITLE
Global inline css is adding padding: 0px to body, how to remove?

### DISCUSSION THREAD

**Discussion by cb911nr** [WordPress/gutenberg#46483] [Q&A]
For some reason either with WordPress 6.1 or the latest gutenberg rc version, our website has the below code inside global inline css…

`body { padding-top: 0px; padding-right: 0px; padding-bottom: 0px; padding-left: 0px; };`

How can I remove this? It’s overwriting our theme’s CSS for body padding, since the global inline css comes after our theme’s CSS.

**Comment by cb911nr** [comment]
As a hacky fix, I was able to just add

wp_dequeue_style( ‘global-styles’ );

To remove the global inline styles which includes these body paddings, but it’s not really a solution

**Comment by luglio7** [comment]
still unanswered. I don't want inline styling when it's useless

**Reply by cb911nr** [reply]
This had solved the problem for us 

`wp_dequeue_style( ‘global-styles’ );`
